website/docs/reference/datasources/querying-mysql.md

```diff
@@ -53,15 +53,39 @@ MySQL databases can be queried using the standard [SQL syntax](https://dev.mysql
 
 ![](/img/postgres.gif)
 
-## Using Prepared Statement (Beta)
 
-Normal query execution simply string concatenates the evaluated values of the javascript bindings to produce the final query. This opens up the possibility of SQL injection by merging untrusted user input with trusted data for execution. Using a prepared Statement is one strategy for mitigating this risk.
 
-Appsmith converts the user query into a parameterized one by replacing the bindings in the query with '?'. The payload is then inserted one by one ensuring that the bindings get properly escaped and sanitized before the query is sent to the database for execution.
+### SQL modes
 
-Follow the guide on [how to use prepared statements](/learning-and-resources/how-to-guides/how-to-use-prepared-statements.md) for efficient and secured data transactions.
+```SQL_MODE``` is a system variable in MySQL that controls the behavior of the MySQL server. It can be used to configure the server to be strict or forgiving when accepting input data, enable or disable standard SQL conformance, or provide better compatibility with other databases. [Strict mode](https://dev.mysql.com/doc/refman/8.0/en/sql-mode.html#sql-mode-strict) can help you ensure the integrity of your data by preventing the database from automatically inserting default values for missing or invalid data. 
 
-## Using Queries in Applications
+#### Setting SQL modes
+
+To change the SQL mode at runtime, you can use a **SET** statement to set the `GLOBAL` or `SESSION` ```sql_mode``` system variable:
+
+```js
+SET GLOBAL sql_mode = 'mode1,mode2,...';
```

Comment:
Switching to the strict mode in this way will require the knowledge of each mode from the user. But there's an easy way to switch to strict mode i.e. `SET sql_mode = TRADITIONAL`. 
Don't you think it comes in handy?

Reply:
@jnikhila @danciaclara should we keep this as a general explanation or add info for a specific mode?

Reply:
If the recommended SQL mode is strict mode then we should mention this.

Reply:
for reference: https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sql-mode-important

Reply:
@harshilp24 - We can add that it's recommended to use strict mode and then link to the official documentation of Mysql?